### DISCUSSION TITLE
Is it possible to simulate some chemical reactions using DeepMD?

### DISCUSSION THREAD

**Discussion by DemingXia** [deepmodeling/deepmd-kit#29] [Q&A] [answered]
I hope to simulate some heterogenous reactions. However, it is to slow to simulate using BOMD. If  the DeepMD model can be used to reproduce some reactions based on some reaction-involved trajectories produced by BOMD?

Thanks~

**Comment by jameswind** [comment, marked as answer]
Sure. Please try it directly. One of the strong advantages of DeePMD is its
ability to describe chemical reactions. If you already have some data, you
could try to fit it with our training code.

**Comment by DemingXia** [comment]
Thank you for your fast reply. I'll try it~
